manuscript/01-automate.txt

```diff
@@ -33,12 +33,12 @@ Then it's applied to the static assets.
 <link rel="stylesheet" type="text/css" href="style.css?v=<?=Assets::REVISION?>">
 ~~~~~~~~
 
-This is truly bad manual steps you could have. It is easy to forget since it requires a code change and if it's forgotten it might break the users' experience serving cached and out dated assets. A manual step where you would have to run a command in connection to the deploy would be better since it would be easier to remember. When you are already have the command line in front of you will be more prone to remember it.
```

Comment:
Wasn't sure what to make of `This is truly bad manual steps you could have` so I rephrased it, hoping to catch your drift